mlflow/store/tracking/rest_store.py

```diff
@@ -171,7 +171,10 @@ def _get_server_version(host_creds: MlflowHostCreds) -> Version | None:
                 endpoint="/version",
                 method="GET",
                 timeout=3,  # Short timeout to fail fast if server version API isn't available
-                max_retries=0,  # No retries - default retry policy takes minutes, which is too long
+                # Disable non-DB SDK retries; default retry policy takes minutes, which is too long
+                max_retries=0,
+                # Disable Databricks SDK retries (0 is interpreted as 'unspecified', so we use -1)
+                retry_timeout_seconds=0.1,
```

Comment:
```suggestion
                retry_timeout_seconds=1.0,
```

Databricks SDK casts this to 0 by `int`.